.code-samples.meilisearch.yaml

```
@@ -335,6 +335,11 @@ search_parameter_guide_matching_strategy_2: |-
   client.index('movies').search('big fat liar', {
     matching_strategy: 'all'
   })
+search_parameter_guide_attributes_to_search_on_1: |-
+  POST 'http://localhost:7700/indexes/movies/search' 
+  with data: { 
+    "q": "adventure", "attributesToSearchOn": ["overview"] 
+  }
```

I am not a maintainer, but I also submitted some code and got the hint to use snake_case.
```suggestion
  client.index('movies').search('adventure', {
    attributes_to_search_on: ['overview']
  })
```

---

Copied from issues, forgot to modify it.
thanks for reminding.

---

> I am not a maintainer, but I also submitted some code and got the hint to use snake_case.

Copied from issues, forgot to modify it.
thanks for reminding.